hastexo/provider.py

```diff
@@ -66,6 +66,9 @@ def init(name):
                 return OpenstackProvider(name, config, sleep_seconds)
             elif provider_type == "gcloud":
                 return GcloudProvider(name, config, sleep_seconds)
+        else:
```

Comment:
I think it's suboptimal that we end up with the same exception type with the same message in two different code paths, as it complicates troubleshooting. Thoughts?

Reply:
Hmm, perhaps yes. I updated the error message now.